exposed-tests/src/main/kotlin/org/jetbrains/exposed/sql/tests/TestUtils.kt

```diff
@@ -10,14 +12,21 @@ fun String.inProperCase(): String = TransactionManager.currentOrNull()?.db?.iden
 
 val currentDialectTest: DatabaseDialect get() = TransactionManager.current().db.dialect
 
-val currentDialectIfAvailableTest: DatabaseDialect? get() =
-    if (TransactionManager.isInitialized() && TransactionManager.currentOrNull() != null) {
-        currentDialectTest
-    } else null
+val currentDialectIfAvailableTest: DatabaseDialect?
+    get() =
+        if (TransactionManager.isInitialized() && TransactionManager.currentOrNull() != null) {
+            currentDialectTest
+        } else null
```

Comment:
Not about the PR, but I'm confused about our Detekt GH action.

Based on the `MultiLineIfElse` setting in `detekt-config`, shouldn't this line be disallowed? It's flagged in the IDE.

Reply:
I would also make an extension for this condition

Reply:
> Not about the PR, but I'm confused about our Detekt GH action.
> 
> Based on the `MultiLineIfElse` setting in `detekt-config`, shouldn't this line be disallowed? It's flagged in the IDE.

This is in the `exposed-tests` module, which is excluded in the detekt GitHub action. I do have it on my to-do list to remove all the detekt warnings in that module at some point and include it in the GitHub action, but I'll create an issue for it.

Reply:
That's on me. I should've realised that code wasn't a part of any datetime module.

Reply:
> I would also make an extension for this condition

I'll work on this in the next PR for refactoring some code in the `exposed-tests` module.